sentry-android-gradle-plugin/src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy

```diff
@@ -347,16 +358,25 @@ class SentryPlugin implements Plugin<Project> {
                     // and run before dex transformation.  If we managed to find the dex task
                     // we set ourselves as dependency, otherwise we just hack outselves into
                     // the proguard task's doLast.
-                    if (dexTask != null) {
-                        dexTask.dependsOn persistIdsTask
-                    } else {
-                        proguardTask.finalizedBy(persistIdsTask)
-                    }
+//                    if (dexTask != null) {
+//                        dexTask.dependsOn persistIdsTask
+//                    } else {
+//                        proguardTask.finalizedBy(persistIdsTask)
+//                    }
+
                     // To include proguard uuid file into aab, run before bundle task.
-                    if (bundleTask != null) {
-                        bundleTask.dependsOn persistIdsTask
+//                    if (bundleTask != null) {
+//                        bundleTask.dependsOn persistIdsTask
+//                    }
+
+                    if (proguardTask != null) {
+                        persistIdsTask.dependsOn proguardTask
+                    }
+
+//                    TODO: do we need to care about dynamic-feature/aka App bundles if we always use the packaging task?
```

Comment:
this should be double-checked, dynamic feature is a new thing and it uses app bundles, so not sure if there are new gradle tasks out there.